tests/virtualization/xen/open_vm_tools.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple vmware client testing with updated open-vm-tools
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
+use xen;
+
+sub run {
+    my ($self) = @_;
+
+    assert_script_run "ssh root\@$_ zypper -n in open-vm-tools" foreach (keys %xen::guests);
+
+    validate_script_output "ssh root\@$_ /usr/bin/vmware-checkvm", sub { m/good/ } foreach (keys %xen::guests);
+
+    assert_script_run "ssh root\@$_ systemctl stop vmtoolsd.service"  foreach (keys %xen::guests);
+    assert_script_run "ssh root\@$_ systemctl start vmtoolsd.service" foreach (keys %xen::guests);
```

Comment:
Why not using `systemctl restart`? Would save one `ssh`-to-all-guests iteration

Reply:
- I just 'translated' what was described in the poo https://progress.opensuse.org/issues/52139
- I guessed that could make more evident if the stop or the start fails especially since the output is not saved due to this "ssh-to-guest thing"

But I am open to just use restart for efficiency